serverless/internal/layout/README.md

```diff
@@ -0,0 +1,86 @@
+Layout
+======
+
+This doc describes the on-disk layout and format of the serverless log files.
+
+> :warning: this is still under development, and the exact layout and contents
+> of the serverless log structure may well change.
+
+Broadly, the layout consists of a self-contained directory hierarchy whose
+contents represents the entire state of the log. With one exception, the
```

Comment:
nit: my first reading of this had "With one exception" following on from the previous statement about self-contained directory. It isn't that. But it could just be me.

Reply:
Flipped around, does that scan better for you?